migration/migration.go

```diff
@@ -170,6 +170,9 @@ func GetMigrations(configuration MigrationConfiguration) Migrations {
 	// Version 20
 	m = append(m, steps{ExecuteSQLFile("020-add-response-mode-to-auth-state-reference.sql")})
 
+	// verion 21
+	m = append(m, steps{ExecuteSQLFile("021-organization-dml.sql")})
```

Comment:
There are a lot of migrations happening here - could we please add some tests to it?

Example: https://github.com/fabric8-services/fabric8-auth/pull/322/files#diff-8300fcfb905de8543555d4fff40a7c2f
The above is an overlap with some of your `identity_role` changes, and also tests whether the database now allows creation of `identity_role` entities with a uuid as the ID / primary key field.

Reply:
I've added some tests for the db changes.